ansible_specdoc/objects.py

```diff
@@ -104,10 +140,49 @@ class SpecReturnValue:
     description: str
     type: FieldType
 
+    returned: str = 'always'
+    version_added: Optional[str] = None
     sample: List[str] = field(default_factory=lambda: [])
+    contains: Optional[Dict[str, 'SpecReturnValue']] = None
```

Comment:
should that Dict type be a string literal? seems that a class literal would fit better